packages/docs/docs/tutorials/data-modeling/family-relationships.md

```diff
@@ -0,0 +1,278 @@
+|  5  | [**Group**](/docs/api/fhir/resources/group) of Patients                                                                                                      | Yes                                                   | Frequently                                    | All Members                                                                                                    | Family Unit                                                           |
+
+- First, you'll have to ask yourself a few questions
+  1. Do family members participate in clinical activiites?
+  2. Is the focus of the treatment an indivdual patient, or a family unit?
+  3. Which family members will get need clinical information (test results, clinical notes, lab specimens), if any?
+  4. How often will Patients share family members?
+
+### Do family members participate in clinical activities?
+
+- If the only reason you need to keep track of family members is for their contact information, and your patients generally don't share family members, then you can use the Patient.contact field (see below). This allows you to
+- However, if family members participate in clinical or billing activities, you can reference their participation by creating a [RelatedPerson](/docs/api/fhir/resources/relatedperson) resource for them (see below).
+- Some examples of how a family member may participate in clinical / billing activities:
+  - Child/dependent is the subject of a clinical office visit, parents / family are in attendance
+  - Family member is the insurance subscriber / payor, patient is a spouse or dependent
+  - A Newborn (patient) is administered a medication by a parent
+
+1.  How often will Patients share family members?
+
+    - Depending on your practice setting, you may have mulitple patients who share family members.
+    - For example, two siblings might be enrolled as separate patients in a pediatric clinic, but they share the same mother and father.
+    - In these situations, it is desireable to store demographic and contact information in single resource, to make it easier to update.
+    - For example, you would want a single resource representing both siblings' father, to make sure his information was always up to date.
+    - However, because a [RelatedPerson](/docs/api/fhir/resources/relatedperson) resource models the relationship between a person and a patient, each sibling would need it's own [RelatedPerson](/docs/api/fhir/resources/relatedperson) resource linking them to the father
+    - Therefore, you will need to model the father as either a[Person](/docs/api/fhir/resources/person) or a Patient, and use either the Patient/[RelatedPerson](/docs/api/fhir/resources/relatedperson)/Person or Patient/[RelatedPerson](/docs/api/fhir/resources/relatedperson)/Patient schemes below to model the family.
+
+2.  Which family members will get need clinical information (test results, clinical notes, lab specimens), if any?
```

Comment:
Add reference to neonatal scenarios here, as hearing test, EBP, PPD and many common workflows are linked to this scenario and make it more comprehensible and you have the general case described.